Odata-docs/includes/appliesto-odataclient-v7.12.1.md

```diff
@@ -0,0 +1,10 @@
+---
+title : "OData client v7.12.1 supported"
+description: OData client v7.12.1 supported
+ms.date: 7/25/2022
+author: gathogojr
+---
+
+# OData client v7.12.1 supported
```

Comment:
Is this supposed to be a heading? This is how it appears rendered on my end:

![image](https://user-images.githubusercontent.com/8460169/182408928-69cfef7b-11f0-42ce-9728-279d1580d14d.png)

Is that how it's supposed to appear?

Reply:
@habbes When I look at other existing doc pages, that's how it appears
![image](https://user-images.githubusercontent.com/28291332/182766363-4d2dd7bf-a7ff-4c3d-95ec-166f6217dcda.png)
![image](https://user-images.githubusercontent.com/28291332/182766404-b50395be-1657-4110-860c-0bb077fe957e.png)